### ISSUE TITLE
Question for M1 chip

### ISSUE THREAD

**Issue by kadir-gunel** [JuliaSIMD/LoopVectorization.jl#317]
Hello, 

 I downloaded Julia(1.7.0 beta-3) targeted for the new Apple M1 chips, and since LoopVectorization is based on pure Julia I wanted to try how it performs on it. I am assuming that it does not need any dependencies on other libraries like intel's one-api. I can download and build the package but when I try to use the package, Julia throws initialization error. Before I share you the full error trace, I wonder that does LoopVectorization package need to perform on avx instructions ? And this new arm instruction sets does not have them? Is this the main reason that I get the error ?  

The ```versioninfo()``` gives:

```Julia
Julia Version 1.7.0-beta3
Commit e76c9dad42 (2021-07-07 08:12 UTC)
Platform Info:
  OS: macOS (arm64-apple-darwin20.5.0)
  CPU: Apple M1
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-12.0.0 (ORCJIT, cyclone)
```

And full error trace is : 

```Julia
ERROR: InitError: UndefVarError: unwrap not defined
Stacktrace:
  [1] redefine_cache(N::Int64)
    @ CPUSummary ~/.julia/packages/CPUSummary/Y1sM6/src/topology.jl:207
  [2] foreach
    @ ./abstractarray.jl:2606 [inlined]
  [3] __init__()
    @ CPUSummary ~/.julia/packages/CPUSummary/Y1sM6/src/CPUSummary.jl:17
  [4] _include_from_serialized(path::String, depmods::Vector{Any})
    @ Base ./loading.jl:768
  [5] _require_search_from_serialized(pkg::Base.PkgId, sourcepath::String)
    @ Base ./loading.jl:854
  [6] _tryrequire_from_serialized(modkey::Base.PkgId, build_id::UInt64, modpath::String)
    @ Base ./loading.jl:783
  [7] _require_search_from_serialized(pkg::Base.PkgId, sourcepath::String)
    @ Base ./loading.jl:843
  [8] _tryrequire_from_serialized(modkey::Base.PkgId, build_id::UInt64, modpath::String)
    @ Base ./loading.jl:783
  [9] _require_search_from_serialized(pkg::Base.PkgId, sourcepath::String)
    @ Base ./loading.jl:843
 [10] _require(pkg::Base.PkgId)
    @ Base ./loading.jl:1097
 [11] require(uuidkey::Base.PkgId)
    @ Base ./loading.jl:1013
 [12] require(into::Module, mod::Symbol)
    @ Base ./loading.jl:997
during initialization of module CPUSummary

```

B.R.

**Comment by chriselrod** [comment]
Sorry, should be fixed by CPUSummary 0.1.2, which [will be released shortly](https://github.com/JuliaRegistries/General/pull/42915).

I won't have an M1 for the next few weeks, so I cannot actually test myself to confirm.

**Comment by chriselrod** [comment]
I'll close this assuming the M1 works again.
